TrevistaOne/src/Components/DrawerComponents/MedicineType.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import {
+  View,
+  Text,
+  ScrollView,
+  StyleSheet,
+  TextInput,
+  Button,
+} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+
+const box = (text, textInput) => {
+  return (
+    <>
+      <Text style={{fontSize: 15, color: '#000000'}}>{text}</Text>
+      <TextInput
+        style={styles.textInputStyles}
+        placeholder={textInput}
+        underlineColorAndroid={'black'}></TextInput>
+    </>
+  );
+};
+
+const MedicineType = () => {
+  return (
+    <ScrollView style={styles.Container}>
+      <View style={styles.MainView}>
+        <LinearGradient
+          style={styles.linearGradient}
+          start={{x: 0, y: 0}}
+          end={{x: 1, y: 0}}
+          colors={['#2AC0EF', '#58D8F9']}>
+          <Text style={styles.MedicineTypeTextStyles}>Medicine Type</Text>
+        </LinearGradient>
+        <View style={styles.Card}>
+          <Text style={{fontSize: 20, color: '#2AC0EF', left: 30}}>
```

Comment:
dont use `inlineStyle` for static styles

Reply:
ok